tap_zendesk/__init__.py

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python3
 import json
-import sys, os
-from datetime import datetime
-from zenpy import Zenpy
+import os
+import sys
```

Comment:
`os` and `sys` are also not used as far as I see.

Reply:
It was only reordered (pylint). It's used [here](https://github.com/Automattic/tap-zendesk/blob/acba7c033d57ff537fa74bd1d044cd35d90cd6a0/tap_zendesk/__init__.py#L112) and [here](https://github.com/Automattic/tap-zendesk/blob/acba7c033d57ff537fa74bd1d044cd35d90cd6a0/tap_zendesk/__init__.py#L77).